itest/asset_meta_test.go

```diff
@@ -0,0 +1,87 @@
+package itest
+
+import (
+	"context"
+	"strings"
+	"testing"
+
+	"github.com/lightninglabs/taproot-assets/taprpc"
+	"github.com/lightninglabs/taproot-assets/taprpc/mintrpc"
+	"github.com/stretchr/testify/require"
+)
+
+// testAssetMeta tests the validation+parsing logic for asset meta data.
+func testAssetMeta(t *harnessTest) {
+	// In this test, we'll attempt to issue the following assets with
+	// distinct meta type. Within each test case, negative failure
+	// scenarios may exist.
+	jsonType := taprpc.AssetMetaType_META_TYPE_JSON
+	testCases := []struct {
```

Comment:
We could test support for unknown meta types here as well.

Reply:
done!